deployment/HDI-Spark/Notebooks/README.md

```diff
@@ -0,0 +1,116 @@
+Deploy Spark .Net REPL to HDInsight Spark cluster with Notebook Experience
+===
+
+# Table of Contents
+This documentation will elaborate the steps on how to deploy Spark .Net REPL to HDI Spark Cluster and submit jobs through Jupyter Notebook using Spark .Net.  
+- [Background](#background)
+- [Pre-requisites](#pre-requisites)
+- [Steps to deploy Spark .Net REPL to HDInsight Spark cluster](#steps-to-deploy-spark-net-repl-to-hdinsight-spark-cluster)
+  - [Step 1. Stop Livy Server](#step-1-stop-livy-server)
+  - [Step 2. Submit HDInsight Script Action](#step-2-submit-hdinsight-script-action)
+  - [Step 3. Start Livy Server](#step-3-start-livy-server)
+   - [Step 4. Set up Spark Default Configs](#step-3-start-livy-server)
+- [Submit Jobs through Jupyter Notebook](#submit-jobs-through-jupyter-notebook)
+
+## Background
+We are currently using [dotnet-try](https://github.com/dotnet/try) as our dotnet REPL. ` dotnet-try ` can be used as a jupyter kernel and we implement it through [livy](https://github.com/apache/incubator-livy) and [sparkmagic](https://github.com/jupyter-incubator/sparkmagic). To enable Jupyter Notebook Experience using Spark .Net, this will include simple manual steps through [Ambari](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-manage-ambari) and submit [script actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux) on HDInsight Spark cluster.
+
+# Pre-requisites:
+Create an [Azure HDInsight spark cluster](https://docs.microsoft.com/en-us/azure/hdinsight/spark/apache-spark-jupyter-spark-sql-use-portal#create-an-hdinsight-spark-cluster) (by visiting the **Azure Portal**, selecting **+ Create a Resource**, and creating a new Azure HDInsight cluster resource), select the **Overview** tab on your cluster. Ensure you've created a cluster with **Spark 2.4** and **HDI 4.0**.
+
+# Steps to deploy Spark .Net REPL to HDInsight Spark cluster
+Open the [Azure portal](https://ms.portal.azure.com/#home). Select **HDInsight Spark cluster** you created in the previous step.
+## Step 1. Stop Livy Server
```

Comment:
Can you make all these manual steps automated by using the Ambari API ?

Reply:
I though there are some permission issue. @imback82 are we planning to investigate using Ambari APIs or we are good for now?

Reply:
I am fine for now and we can revisit (in a separate PR if needed) unless @suhsteve already has a working solution to share.

Reply:
Sure we can revisit this if needed.